packages/candid-ui/README.md

```diff
@@ -0,0 +1,336 @@
+## Candid UI
+
+This package provides a web component that can generate an interactive UI for a provided canister. It can be used to interact with canisters that have a Candid interface.
+
+### Usage
+
+The web component is called `candid-ui`. In its basic usage, it takes a `canisterId` attribute, which is the canister ID of the canister to interact with. If no `canisterId` is provided, it will render a text input to allow the user to enter a canister ID. The component will also automatically infer a `host` from the current URL, either selecting a commonly used localhost port if present or mainnet, but this can be overridden by providing a `host` attribute.
+
+### Installation
+
+To use the component, you can either install it from npm:
+
+```bash
+npm install @dfinity/candid-ui
+```
+
+However, the package is optimized to be used in a script tag, so we recommend using the auto setup script from unpkg:
+
+```html
+<script type="module" src="https://unpkg.com/@dfinity/candid-ui/dist/auto.js"></script>
```

Comment:
Using a CDN introduces a centralized component to a FE app. CDNs are entirely outside of our control, they can pull packages (which is extremely unlikely to be fair) or track users. I'm not against providing documentation for how to use it, but I'm not sure that we should be saying "we recommend".

Reply:
We could publish it to a canister "CDN" where people could pull it from

Reply:
For now I changed the language. I'll go ahead and merge and publish, so we can get the base release shipped

Reply:
I think a canister CDN is a great idea!